src/Site.js

```diff
@@ -1,4 +1,4 @@
-const cheerio = require('cheerio');
+const cheerio = require('cheerio'); require('./lib/markbind/src/patches/htmlparser2');
```

Comment:
added this in all files to ensure unit tests load the patch, which is more than ever important with the extra 2 "patches" for `recognizeSelfClosing` and `lowerCaseAttributeNames`